acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueGetSignersRpcTest.java

```diff
@@ -18,53 +18,53 @@
 import tech.pegasys.pantheon.tests.acceptance.dsl.node.PantheonNode;
 
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
 
-@Ignore
 public class CliqueGetSignersRpcTest extends AcceptanceTestBase {
   private PantheonNode minerNode1;
   private PantheonNode minerNode2;
   private PantheonNode minerNode3;
   private PantheonNode[] allNodes;
-  private PantheonNode[] initialNodes;
 
   @Before
   public void setUp() throws Exception {
-    final String[] validators = {"miner1", "miner2"};
+    final String[] validators = {"miner1"};
     minerNode1 = pantheon.createCliqueNodeWithValidators("miner1", validators);
     minerNode2 = pantheon.createCliqueNodeWithValidators("miner2", validators);
     minerNode3 = pantheon.createCliqueNodeWithValidators("miner3", validators);
-    initialNodes = new PantheonNode[] {minerNode1, minerNode2};
     allNodes = new PantheonNode[] {minerNode1, minerNode2, minerNode3};
     cluster.start(allNodes);
   }
 
   @Test
   public void shouldBeAbleToGetValidatorsForBlockNumber() {
-    cluster.verify(clique.validatorsAtBlockEqual("0x0", initialNodes));
-    cluster.waitUntil(wait.chainHeadIsAt(1));
-    minerNode1.execute(cliqueTransactions.createAddProposal(minerNode3));
-    minerNode2.execute(cliqueTransactions.createAddProposal(minerNode3));
+    cluster.verify(clique.validatorsAtBlockEqual("0x0", minerNode1));
+    minerNode1.waitUntil(wait.chainHeadIsAt(1));
 
+    minerNode1.execute(cliqueTransactions.createAddProposal(minerNode2));
     cluster.waitUntil(wait.chainHeadHasProgressed(minerNode1, 1));
-    cluster.verify(clique.validatorsAtBlockEqual("0x2", initialNodes));
+    cluster.verify(clique.validatorsAtBlockEqual("0x2", minerNode1, minerNode2));
 
-    minerNode1.waitUntil(wait.chainHeadHasProgressed(minerNode1, 1));
-    cluster.verify(clique.validatorsAtBlockEqual("0x3", allNodes));
+    minerNode1.execute(cliqueTransactions.createAddProposal(minerNode3));
```

Comment:
these lines probably don't add a lot of value - they can be removed (I suspect they do not increase coverage ... so .... remove?)

Reply:
Agreed. Don't think they do and could make the test more brittle too.
Removed them